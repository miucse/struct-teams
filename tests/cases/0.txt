3

Brutal Bytes
Manarat International University
6
Shamim Ahmed
Jakariya Abbas
Ahmad Hasan Mubashshir

Printf("hello");
Manarat International University
8
Mostakim Sakib
Irfan Khan
Saidur Rahman

team X
Manarat International University
5
Muhammad hasnat
Raihan Chowdhury
Sakib Al-Hashem
