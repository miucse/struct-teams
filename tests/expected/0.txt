Team name  : Printf("hello");
Institution: Manarat International University
Solved     : 8
   Member 1: Mostakim Sakib
   Member 2: Irfan Khan
   Member 3: Saidur Rahman
